src/nvim/getchar.c

```diff
@@ -4237,3 +4238,17 @@ static bool typebuf_match_len(const uint8_t *str, int *mlen)
   *mlen = i;
   return str[i] == NUL;  // matched the whole string
 }
+
+/// Retrieve the mapblock at the index either globally or for a certain buffer
+///
+/// @param  index  The index in the maphash[]
+/// @param  buf  The buffer to get the maphash from. NULL for global
+mapblock_T *get_maphash(int index, buf_T *buf)
+    FUNC_ATTR_PURE
+{
+  if (index > MAX_MAPHASH) {
```

Comment:
The comparison should be `>=`, otherwise an index of `MAX_MAPHASH (= 256)` could access an element **after**
the array `maphash`.

Reply:
Thanks @oni-link ! Fixed on master.